streams/src/test/java/org/apache/kafka/test/MockInternalNewProcessorContext.java

```diff
@@ -212,6 +214,7 @@ public String changelogFor(final String storeName) {
         return "mock-changelog";
     }
 
+    // TODO check if need to add detail on forwarding
```

Comment:
I don't think so. It's just doing the same thing that the main implementation does.